gdk-pixbuf/src/pixbuf.rs

```diff
@@ -90,8 +90,6 @@ impl Pixbuf {
     pub fn from_file<T: AsRef<Path>>(filename: T) -> Result<Pixbuf, Error> {
         #[cfg(not(windows))]
```

Comment:
This will fail to compile now. You'll have to remove that `#[cfg]`.

But also *most* of these functions can be autogenerated now (remove them from `Gir.toml` for that).

Reply:
only that one `#[cfg]`?. Should I remove the whole file for `Gir.toml`? Correct me if I'm wrong but this is the toml file that generates the functions. So If I remove them what purpose will the files serve?

Reply:
The autogenerated code is in `src/auto`, this is manually written. You have to remove the `#[cfg]` because otherwise this won't compile on Windows.

But as written in the other comment, this function can be autogenerated now so you can also remove the function completely (and if the file is empty afterwards you can remove the `src/pixbuf.rs` file too).